src/Blazored.LocalStorage/BrowserStorageProvider.cs

```diff
@@ -73,6 +77,15 @@ public void RemoveItem(string key)
             _jSInProcessRuntime.InvokeVoidAsync("localStorage.removeItem", key);
```

Comment:
I know this wasn't part of your work but this is wrong so it makes sense to get it fixed
```suggestion
            _jSInProcessRuntime.InvokeVoid("localStorage.removeItem", key);
```